extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegeObject.scala

```diff
@@ -71,7 +72,7 @@ object PrivilegeObject {
       actionType,
       table.database.orNull,
       table.table,
-      columns,
+      columns.filter(!HudiUtils.isHudiMetaField(_)),
```

Comment:
So if the table fields happens to be the same as meta field names, are the columns getting away from privilege checks?

Reply:
> So if the table fields happens to be the same as meta field names, are the columns getting away from privilege checks?

good question....,maybe we need to check if it's a hudi table.

Reply:
Yes, if we have to implement it this time. Make sure the clarification are mentioned in PR description.

Reply:
@yaooqinn What do you think about this? Iceberg also have similar problem.

Reply:
Checked iceberg won't have such problem...
```

"Permission denied: user [someone] does not have [select] privilege on [iceberg_ns/table1/id,iceberg_ns/table1/name,iceberg_ns/table1/city]" did not contain "xxxx"
ScalaTestFailureLocation: org.apache.kyuubi.plugin.spark.authz.ranger.IcebergCatalogRangerSparkExtensionSuite at (IcebergCatalogRangerSparkExtensionSuite.scala:373)

```